pages/star-search/index.tsx

```diff
@@ -52,13 +52,27 @@ type StarSearchChat = {
 export default function StarSearchPage({ userId, bearerToken, ogImageUrl }: StarSearchPageProps) {
   const [starSearchState, setStarSearchState] = useState<"initial" | "chat">("initial");
   const [chat, setChat] = useState<StarSearchChat[]>([]);
-  const [input, setInput] = useState("");
   const inputRef = useRef<HTMLInputElement>(null);
   const [isRunning, setIsRunning] = useState(false);
   const isMobile = useMediaQuery("(max-width: 576px)");
   const [showSuggestions, setShowSuggestions] = useState(false);
   const scrollRef = useRef<HTMLDivElement>(null);
 
+  function addPromptInput(prompt: string) {
+    if (!inputRef.current?.form) {
+      return;
+    }
+
+    inputRef.current.value = prompt;
+    const { form } = inputRef.current;
+
+    if (typeof form.requestSubmit === "function") {
```

Comment:
form.submit() wasn't working but not with the [synthetic events in React](https://react.dev/reference/react-dom/components/common#react-event-object), so I went with this. See https://github.com/facebook/react/issues/6796#issuecomment-594081509